server/prisma/schema.prisma

```diff
@@ -8,28 +8,121 @@ datasource db {
   url      = "file:./dev.db"
 }
 
-// This comes with the boilerplate.
 model User {
-  id                  String          @id @default(cuid())
-  createdAt                        DateTime        @default(now())
-  updatedAt                        DateTime        @updatedAt
-  email           String  @unique
-  password      String
-  role          String
-  lastLogin     DateTime?  
-  // LL: https://www.smashingmagazine.com/2017/11/safe-password-resets-with-json-web-tokens/
-  resetPasswordToken String
-  dateResetPasswordRequest DateTime?
-  validateEmailToken String
-  isEmailValidated Boolean
-  name  String
+  id                         Int                         @id @default(autoincrement())
+  fullName                   String?
+  email                      String                      @unique
+  password                   String
+  feeds                      Feed[]
+  boards                     Board[]
+  userSourceFavorites        UserSourceFavorite[]
+  userActivatedSubscriptions UserActivatedSubscription[]
+  createdAt                  DateTime                    @default(now())
+  setting                    Setting?
+
+  @@map("users")
 }
 
-model Feed {
+// user settings like theme
+
+model Setting {
+  id     Int    @id @default(autoincrement())
+  user   User   @relation(fields: [userId], references: [id])
+  userId Int    @unique
+  /// light or dark
+  theme  String
+
+  @@map("settings")
+}
+
+// integrations
+
+model SubscriptionSpec {
+  id                        Int                         @id @default(autoincrement())
+  /// FREE / PRO / TRIAL
+  type                      String
+  priceInCents              Int
+  description               String
+  createdAt                 DateTime                    @default(now())
+  updatedAt                 DateTime                    @updatedAt
+  userActivatedSubscription UserActivatedSubscription[]
+
+  @@map("subscriptions")
+}
+
+/// subscriptions which are were bougth by User
+model UserActivatedSubscription {
+  id             Int              @id @default(autoincrement())
+  createdAt      DateTime         @default(now())
+  user           User             @relation(fields: [userId], references: [id])
+  userId         Int
+  subscription   SubscriptionSpec @relation(fields: [subscriptionId], references: [id])
+  subscriptionId Int
+  expiredAt      DateTime
+
+  @@map("user_activated_subscriptions")
+}
+
+model Post {
+  id        Int      @id @default(autoincrement())
+  title     String
+  content   String
```

Comment:
one thing to consider here is whether HTML is allowed in the content (in an RSS reader it would be), in which case you'd want to rename the field contentHtml or add a comment above it indicating that this field is expected to take html.